Spigot-Server-Patches/0700-Add-New-Logo.patch

```diff
@@ -0,0 +1,8048 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: OskarZyg <56176746+OskarZyg@users.noreply.github.com>
+Date: Wed, 31 Mar 2021 19:34:11 +0100
+Subject: [PATCH] Add New Logo
+
+
+diff --git a/src/main/resources/logo.png b/src/main/resources/logo.png
+index a7d785f60c884ee4ee487cc364402d66c3dc2ecc..25ebce7707d1290717574c753f6ba1b2be82cc07 100644
+GIT binary patch
+literal 404147
+zcmV*?KrO$CP)<h;3K|Lk000e1NJLTq00O)K00O)S1^@s64Z3VY001BWNkl<ZcmeF4
+z1yohp+r~e{nfW_o3rKf&OP7=hih$jTomhz7-HC+(7NS^4h++VWirt+v&NyR?&bXd$
+zy@xq?x!em};1(R#+H0S4?umW&-TT=;-uv!%|6f8a1#pS01Fj8lZ9v*>AU+@nUgoyA
+zbL}qPy!@@ybpQ54MB2L|x~V#+Uz~=^O&i$Ts6NuYn&D!<&bT>mBJRa4#J$8-DA>Ci
+zd5Oz$HgPG=>{*7hdzYi&_+~u4kOVx-1@H;nU)eQqZ1XJSEEq30?Lpd_ThpHZe>Lre
+zx%cJ3j%x!l*aj38(JeRx>*g;j6?9n;9@nKe259PW0~-gsGS>zUs;MHYg&ztg4abeO
+zb8vavBAnm07#W-A;?TP3NM1h;C$`PQg`_ojbap?!KE43pkFsfwJEPPZxR;fHw5@Yx
+zu03$$jqHp!9Pn}j1}6|)8<2V%*fMAsTD$s`3wGy4PeGWjF}5ozO1<3NU?w|*nIyl9
+zWKB(Ew++J8Nh5H1^=xEsn2!_dW@6up80=USja^H|<Iskg$T_$HuPz@4KHMu6*0>KV
+zTLZu0z?xXxS-F(c1a3SN+6Fk_<r;_3#v#c8gnL%a1PA)w_cdyW7LM*Xvgc5lz<YG>
+zAvD)B!Nwnd<k~H#U1Dtm1hrYge#jd&5P3_dBYove>{~n;n`Vr_y6MBParOx8S~>}7
+zJC@-|ZVK?%gUV7HcYOtG;Aq?|{GO8{*7m@SQSv`Valp$B8k|UQZ9uARU{(Ji=-<9`
+znXvol?qh^l+GBRZCQ>agH}J{-z{mR?rdXOFqf;nOO&WpY^Cx5H%rRItegGCl_rvl@
+zgRo)7C>&fj8<&pmz^A8|D~M#=W#X!VyQlZy?C#~9CUE0enQedrUapap*+_T?2X4E1
+z4ffLhKKB?K!Q0#x=hCyw1m3wrMq_{$=eH~E?@S(_zpKvwkmO{Clx`6?JZ>oBCl15n
+zQGGFEa97M2+6@cF_Q$#zqmZy-2F@jI!9Q>Bh>Op-`&MZUyu-nD({X#rLT()6Sf<Q2
+zzyUAUNXl#^@;Jb8WtIal|DiS(4<Co2oqOSJ8BrNBQ%=HH(*Rp)*XG(Qr&$8q!0y_L
+zh*xQX-CnNP)hz-WqIzS_fUcO-GYsQ;hG9a#2+SDP1IuHEW9O1-$WGdZk1uXiDSmM`
+zlWYwf+Atfxo=q0m9=I_;4#ofpYZ(W;60UHr6LOu9YXgOC;8=JhJWQ-``NHKgt-Fyf
+z-k8=zg&Wv7y+ZsiKVzez9@Ydl$AZZAn9`{|hK034RJ$My4sVUIy~8nMSYNE0F&+oj
+zFTsto$AQnUB}-!5)5Kl_x3l&lbL(PG6F5yMcN^e<muoELZY&(Aaj;%hZD3E`x|lOO
+z8qs|RmucNyH+vDfXc=(rv<7L__Buadjg}_n+uC4~zaIt%`lDB%54tq>K#xEl^luZ4
+zu{}Fs;rJ2QIByzG?B0SW1r*+k`>Po>@CFIXCgaZXd7LiPAYG{J_`?A&*H}2g;M#!1
```

Comment:
```suggestion
zzyUAUNXl#^@;Jb8WtIal|DiS(4<Co2oqOSJ8BrNBQ%=HH(*Rp)*XG(Qr&$8q!0y_L
zh*xQX-CnNP)hgdgf;;fdgs-g-gr--wkk,,vlaaaaaaaweW9O1-$WGdZk1uXiDSmM`
zlWYwf+Atfxo=q0m9=I_;4#ofpYZ(W;60UHr6LOu9YXgOC;8=JhJWQ-``NHKgt-Fyf
z-k8=zg&Wv7y+ZsiKVzez9@Ydl$AZZAn9`{|hK034RJ$My4sVUIy~8nMSYNE0F&+oj
zFTsto$AQnUB}-!5)@@@$$$%%%tt(PG6F5yMcN^e<muoELZY&(Aaj;%hZD3E`x|lOO
z8qs|RmucNyH+vDfXc=(rv<7L__Buadjg}_n+uC4~zaIt%`lDB%54tq>K#xEl^luZ4
zu{}Fs;rJ2QIByzG?B0SW1r*+k`>xdffgt4445hjkjmcsd7LG{J_`?A&*H}2g;M#!1
```

Reply:
This violates our code style, please fix so we can merge asap.

Reply:
What exactly is being violated?